packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/translations/Messages_fr_FR.json

```diff
@@ -18,5 +18,43 @@
+  "ip_agora_ipv6_option_header": "Dans de nombreux cas, vous aurez peut-être besoin d'une IPv4 standard, la plus utilisée, ou d’une nouvelle version du protocole : IPv6. Veuillez noter que la liste des produits compatibles peut varier.",
+  "ip_agora_ipv6_option_tile_title": "Nouveau préfixe /56 IPv6 (attribué par OVHcloud)",
+  "ip_agora_ipv6_option_tile_description": "services. Obtenez votre nouveau préfixe IPv6 à utiliser dans les services vRack.",
+  "ip_agora_ipv6_list_continent_all": "Toutes les localisations",
+  "ip_agora_ipv6_location_eu-west-par": "Europe (France - Paris)",
+  "ip_agora_ipv6_location_eu-west-gra": "Europe (France - Gravelines)",
+  "ip_agora_ipv6_location_eu-west-rbx": "Europe (France - Roubaix)",
+  "ip_agora_ipv6_location_eu-west-sbg": "Europe (France - Strasbourg)",
+  "ip_agora_ipv6_location_eu-west-lim": "Europe (Germany - Limburg)",
+  "ip_agora_ipv6_location_eu-central-waw": "Europe (Poland - Warsaw)",
+  "ip_agora_ipv6_location_eu-west-eri": "Europe (UK - Erith)",
+  "ip_agora_ipv6_location_us-east-vin": "North America (US - East - Vinthill)",
+  "ip_agora_ipv6_location_us-west-hil": "North America (US - West - Hillsboro)",
+  "ip_agora_ipv6_location_ca-east-bhs": "North America (Canada - East - Beauharnois)",
+  "ip_agora_ipv6_location_ap-southeast-sgp": "Asia Pacific (Singapore - Singapore)",
+  "ip_agora_ipv6_location_ap-southeast-syd": "Asia Pacific (Australia - Sydney)",
+  "ip_agora_ipv6_location_eu-west-rbx-snc": "Europe (France - Roubaix) (snc)",
+  "ip_agora_ipv6_location_eu-west-sbg-snc": "Europe (France - Strasbourg) (snc)",
+  "ip_agora_ipv6_location_ca-east-tor": "North America (Canada - East - Toronto)",
+  "ip_agora_ipv6_location_ap-south-mum": "Asia Pacific (India - Mumbai)",
+  "ip_agora_ipv6_location_labeu-west-1-preprod": "LABEU (France - Croix) (preprod)",
+  "ip_agora_ipv6_location_labeu-west-1-dev-2": "LABEU (France - Croix) (dev-2)",
+  "ip_agora_ipv6_location_labeu-west-1-dev-1": "LABEU (France - Croix) (dev-1)",
+  "ip_agora_ipv6_location_eu-west-lz-bru": "Europe (Belgium - Brussels) (lz)",
+  "ip_agora_ipv6_location_eu-west-lz-mad": "Europe (Spain - Madrid) (lz)",
+  "ip_agora_ipv6_location_eu-west-gra-snc": "Europe (France - Gravelines) (snc)",
+  "ip_agora_ipv6_location_us-east-lz-dal": "North America (US - East - Dallas)",
+  "ip_agora_ipv6_location_us-west-lz-lax": "North America (US - West - Los Angeles)",
+  "ip_agora_ipv6_location_us-east-lz-chi": "North America (US - East - Chicago)",
+  "ip_agora_ipv6_location_us-east-lz-nyc": "North America (US - East - New York)",
+  "ip_agora_ipv6_location_us-east-lz-mia": "North America (US - East - Miami)",
+  "ip_agora_ipv6_location_us-west-lz-pao": "North America (US - West - Palo Alto)",
+  "ip_agora_ipv6_location_us-west-lz-den": "North America (US - West - Denver)",
+  "ip_agora_ipv6_location_us-east-lz-atl": "North America (US - East - Atlanta)",
+  "ip_agora_ipv6_location_eu-west-lz-mrs": "Europe (France - Marseille)"
```

Comment:
Suggestion : most of those keys are present elsewhere in the manager (bm-server-components/general-infos or cluster/dashboard/general-infos). Could be nice to think of a way to mutualize

Reply:
I agree with you, but I think we should do it in a separate ticket